data/mt/legislature/David-Bedey-8583c5f0-fc55-4f5b-9664-19d44f7f25b3.yml

```diff
@@ -3,23 +3,24 @@ name: David Bedey
 given_name: David
 family_name: Bedey
 gender: Male
-email: David.Bedey@mtleg.gov
+email: David.Bedey@legmt.gov
 party:
 - name: Republican
 roles:
 - type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
   district: '86'
+offices:
```

Comment:
@bfossen-ce this seems to be the case for many records so I won't tag you on each but just wanted to flag it looks like we are getting rid of an office address (22-25) but not replacing it with an address (13). Not sure if this is ok